exe/fil-tools/cmd/fpa_add.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"os"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	fpaAddCmd.Flags().StringP("token", "t", "", "FPA access token")
+
+	fpaCmd.AddCommand(fpaAddCmd)
+}
+
+var fpaAddCmd = &cobra.Command{
+	Use:   "add [cid|path]",
```

Comment:
Decided to combine adding by cid and path into this single command. Also using args now for the cid/path instead of flags.

Reply:
Would be nice to handle stdin at some point. I was thinking the same for textile.